core/src/main/scala/jio/package.scala

```diff
@@ -134,10 +153,35 @@ package object jio extends DecorateAsScala with DecorateAsJava {
   )
 
   def file(s: String, ss: String*): File        = ss.foldLeft(new File(s))(new File(_, _))
-  def path(s: String, ss: String*): Path        = ss.foldLeft(jnf.Paths get s)(_ resolve _)
+  def path: String => Path                      = jnf.Paths get _
```

Comment:
Type inference, the difference between `def x(x:X): Y` and `def x: X => Y` and it was not used with more than one argument.

I did not change file because I wanted to keep the changes to a minimum. Happy to change file as well.